### PR TITLE
Frontend Web Development CSS ANIMATION

### LINKED ISSUE #1056: CSS Animations in Documentation
<hr>

## Description 📜

Different kind of animations in css
Different technique to do animations of different kind using Examples

<hr>

## Domain of Contribution 📊

- [X] Frontend Dev (HTML, CSS, JS)

<hr>

### PR DESCRIPTION
**Pull request by priyanka001tech**
<hr>

## Description 📜

Please include a summary of the change and which issue is fixed. Please also include relevant motivation and context. 
Fixes #1056 
<h2>Add documentation about CSS Animation</h2> 
In the Road-map of GWOC I saw a topic named CSS animation so I have made a documentation for that which includes different properties of animation and live examples also. 

<hr>

## Type of change 📝

- [x] Documentation (Content Creation in the form of codes or tutorials)

<hr>

## Domain of Contribution 📊

- [x] Frontend Dev HTML/CSS/JS

<hr>

## Checklist ✅

- [x] I follow [Contributing Guidelines](https://github.com/girlscript/winter-of-contributing/blob/main/.github/CONTRIBUTING.md) & [Code of conduct](https://github.com/girlscript/winter-of-contributing/blob/main/.github/CODE_OF_CONDUCT.md) of this project.
- [x] I have performed a self-review of my own code or work.
- [x] I have commented my code, particularly in hard-to-understand areas.
- [x] My changes generates no new warnings.
- [x] I'm GWOC'21 contributor

<hr>

<hr>

### CODE REVIEW

**Comment by priyanka001tech** [comment]
Hello @ghulamyazdani @code-reaper08 , Pls check this once and let me know if I need to do some fixes or its fine

File: Web_Development/Javascript/CSS_ANIMATION/README.md [hunk 1]
```diff
@@ -0,0 +1,129 @@
+#**CSS ANIMATION :--**
```
Comment (code-reaper08) [inline Web_Development/Javascript/CSS_ANIMATION/README.md:1]: Getting `#` here in live file too. Try correcting this.

**Review by code-reaper08 — CHANGES_REQUESTED** [review]
Kindly look into the directory structure. You are pushing these into `Web_Development/Javascript/CSS_ANIMATION/README.md`

@ghulamyazdani have a look here.

**Comment by priyanka001tech** [comment]
@code-reaper08  I have solved the issue please check CSS Animation added #1969

**Pull request closed without merging**